tests/integration/query/latest_commits/simple_test.go

```diff
@@ -20,7 +20,7 @@ import (
 // desired behaviour (should return all latest commits).
 func TestQueryLatestCommits(t *testing.T) {
 	test := testUtils.TestCase{
-		Description: "Simple latest commits query",
+		Description: "Simple latest commits queries",
```

Comment:
question: why is this plural now? I see only one query request in the test. Also then test name itself has only `Query` in it.

Reply:
honestly, fantastic question 🙃. I just saw fred suggestion and went with it. Im thinking now that the `s` was there accidentally from me trying to press `ctrl+s` to save at some point and not registering the `ctrl`

Reply:
I assumed John was meaning to make it plural because there were multiple queries in the test (Git change was not showing the full test and just took it for granted)

The assumption cascade is strong with this one... lol

Reply:
Lol 😆 I looked at it three times, thinking am I missing something or tripping.